package.json

```diff
@@ -22,6 +21,13 @@
     "build": "gulp build",
     "dist": "gulp dist"
   },
+  "files": [
```

Comment:
I don't think, I can add inline comments.
I would prefer to mention here that `npm publish` picks up only these resources to be included in tarball.

Reply:
There should be license as well, right?
Also, as we learned, there's no value to publish `src`. Instead we just need to put a compilable `activities-es6.js` with `module.exports` and bundled `activities.min.js`.

Reply:
Done.
Had to create `./index.js`, and the gulp task will use this file to generate `./index-es6.js`, which is being referenced in the `main` attribute of package.json